### PR TITLE
Update About Us 

### PR DESCRIPTION
**Pull request by rowa97**

### CODE REVIEW

File: netlify/pages/about.yaml [hunk 1]
```diff
@@ -82,6 +85,9 @@ team:
   - image: /images/mateusz_wit_headshot.png
     name: Mateusz Wit
     role: Software Engineer
+  - image: /images/harrison-yeager-headshot.jpg
+    name: Harrison Yeager
+    role: Ops
```
Comment (ssspear) [inline netlify/pages/about.yaml]: We've got the room for 'Operations' so I'd say go with it ??

File: netlify/pages/about.yaml [hunk 2]
```diff
@@ -27,7 +27,10 @@ team:
     role: Software Engineer
   - image: /images/faust_headshot.png
     name: Scott Faust
-    role: Growth & Ops
+    role: Growth
+  - image: /images/deborah-garnel-headshot.jpg
+    name: Deborah Garnel
+    role: SDET
```
Comment (ssspear) [inline netlify/pages/about.yaml]: Deb will be the only person with an acronym for a title. We cool with that? :)

File: netlify/pages/about.yaml (hunk 1 above)
Comment (rowa97) [reply netlify/pages/about.yaml]: too long for the card space unfortunately

File: netlify/pages/about.yaml (hunk 2 above)
Comment (rowa97) [reply netlify/pages/about.yaml]: we also have Marc with CEO

Comment (ssspear) [reply netlify/pages/about.yaml]: OH right, my bad.

**Pull request merged**